src/main.rs

```diff
@@ -170,8 +170,8 @@ fn main() -> Result<(), Box<dyn Error>> {
       log_info!("Successfully added record: {:?}", record);
     }
     Commands::Add { config, interactive, domain, query_type, class, ttl, host, ip, priority } if !interactive => {
-      log_info!("Loading from config file '{}'...", config);
       let settings = DnsSettings::load(config.clone())?;
+      log_info!("Loaded from config file '{}'...", settings.config_file);
```

Comment:
These log messages can be moved into the `DnsSettings::load` function and then we shouldnt need to store the config file path in the config file

Reply:
Done in 4f0427e269accb2957e43635f51d696435acc506